dp/cloud/go/services/dp/storage/cbsd_manager.go

```diff
@@ -217,18 +249,39 @@ func (c *cbsdManagerInTransaction) updateCbsd(networkId string, id int64, data *
 		Update()
 }
 
-func (c *cbsdManagerInTransaction) checkIfCbsdExists(networkId string, id int64) error {
-	_, err := db.NewQuery().
+func (c *cbsdManagerInTransaction) enodebdUpdateCbsd(data *DBCbsd) error {
+	mask := db.NewIncludeMask("id")
```

Comment:
This should be a broader mask, so that comparison of installation params is possible

Reply:
as agreed, will do in a later pr